unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/WearableController.cs

```diff
@@ -156,8 +156,9 @@ public void SetAnimatorBones(SkinnedMeshRenderer skinnedMeshRenderer)
         boneRetargetingDirty = false;
     }
 
-    public void CleanUp()
+    public virtual void CleanUp()
```

Comment:
Why you needed to make this function as virtual?

Reply:
Override in `BodyShapeController`